cloud_utils/storage/azure.py

```diff
@@ -36,6 +36,9 @@ def _sign_params(key: str, params: dict):
 def head_headers_and_url(bucket_name: str, blob_name: str):
     now = datetime.datetime.utcnow().strftime("%a, %d %b %Y %H:%M:%S GMT")
     config = _get_connection_config()
+    account_name_key = "AccountName\\"
```

Comment:
Why you have \\? can we remove it from the dict?

Reply:
we can